unattended_installer/install_functions/installMain.sh

```diff
@@ -334,6 +335,7 @@ function main() {
         filebeat_configure
         installCommon_startService "filebeat"
         common_logger "--- Wazuh dashboard ---"
+        dashboard_installDependencies
```

Comment:
In the AIO case, maybe all dependencies should be installed at the beginning

Reply:
In that case, if a user cancels the installation before the Wazuh dashboard is installed, its dependencies will keep installed in the system. For these types of examples, I thought it would be better to install the dashboard dependencies in its installation stage.

Reply:
Yeah, it makes sense.